__tests__/integration.test.js

```diff
@@ -28,10 +29,52 @@ describe("GET", () => {
           });
         });
     });
+
     test("response is 404 if endpoint doesn't exist", () => {
-        return request(app)
-          .get("/api/notAnEndpoint")
-          .expect(404)
-      });
+      return request(app).get("/api/notAnEndpoint").expect(404);
+    });
+  });
+
+  describe("/api/articles/:article_id", () => {
+    test("responds with 200 upon success", () => {
+      return request(app).get("/api/articles/1").expect(200);
+    });
+
+    test("responds with article object with expected keys", () => {
+      const isoDate = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z$/;
+      return request(app)
+        .get("/api/articles/2")
+        .then((response) => {
+          expect(response.body.article).toMatchObject({
```

Comment:
Great use of the flexible jest matcher toMatchObject() here.

For this endpoint, we know exactly which article we expect to return. Therefore you should be expecting the exact values for article 2 rather than the data types here.